tests/rptest/clients/types.py

```diff
@@ -40,19 +40,17 @@ class TopicSpec:
     TIMESTAMP_CREATE_TIME = "CreateTime"
     TIMESTAMP_LOG_APPEND_TIME = "LogAppendTime"
 
-    def __init__(
-            self,
-            *,
-            name=None,
-            partition_count=1,
-            replication_factor=3,
-            cleanup_policy=CLEANUP_DELETE,
-            compression_type=COMPRESSION_PRODUCER,
-            message_timestamp_type=TIMESTAMP_CREATE_TIME,
-            segment_bytes=1 * (2 ^ 30),
-            retention_bytes=None,
-            retention_ms=None
-    ):
+    def __init__(self,
+                 *,
+                 name=None,
+                 partition_count=1,
+                 replication_factor=3,
+                 cleanup_policy=CLEANUP_DELETE,
+                 compression_type=COMPRESSION_PRODUCER,
+                 message_timestamp_type=TIMESTAMP_CREATE_TIME,
+                 segment_bytes=1 * (2 ^ 30),
+                 retention_bytes=None,
+                 retention_ms=None):
```

Comment:
thanks for separating the white space commits. it throws a wrench in git blame, but I'll take reviewability instead!